website/templates/emails/digest_reviews_moderators.html.mako

```diff
@@ -23,6 +39,9 @@
             </tr>
         </tbody>
     </table>
+    % if provider_type == 'registration':
+        You are receiving these emails because you are a moderator on {provider.name}. To change your moderation notification preferences, visit your <a href="https://osf.io/settings/notifications/"> notification settings.</a>
```

Comment:
This chunk comes from here: https://github.com/CenterForOpenScience/osf.io/blob/58e85d1905aac115026e4e851747aff7c7975440/website/templates/emails/notify_base.mako#L53

Should be covered by your correction of `notification_settings_url` in tasks.py

Reply:
👍 Removed